docs/CHANGELOG.asciidoc

```diff
@@ -61,6 +61,11 @@ are unexpectedly missing. (See {ml-pull}721[#721].)
 * Trap numeric errors causing bad hyperparameter search initialisation and repeated
 errors to be logged during boosted tree training. (See {ml-pull}732[#732].)
 
+=== Bug Fixes
+
+* Restore from checkpoint could damage seasonality modelling. For example, it could
```

Comment:
For release notes we have to use the US spelling of modeling with one 'l'.

Reply:
See [2e9d47c](https://github.com/elastic/ml-cpp/pull/821/commits/2e9d47cd0955a5a8c22509e8c239e50438b3c5e0)